pkg/crc/machine/vsock.go

```diff
@@ -113,6 +113,12 @@ func vsockPorts(preset crcPreset.Preset) []types.ExposeRequest {
 				Local:    fmt.Sprintf("%s:%d", localIP, cockpitPort),
 				Remote:   fmt.Sprintf("%s:%d", virtualMachineIP, cockpitPort),
 			})
+		exposeRequest = append(exposeRequest,
+			types.ExposeRequest{
+				Local:    constants.GetDockerSockOnHost(),
+				Remote:   fmt.Sprintf("ssh-tunnel://%s//var/run/docker.sock?user=core&key=%s", virtualMachineIP, constants.GetPrivateKeyPath()),
+				Protocol: "unix",
```

Comment:
The other entries list `Protocol` first, better to be consistent. This could be merged with the `append` call right before it.

Reply:
This is only half resolved.

Reply:
Resolved.